test/ruby/test_compile_prism.rb

```diff
@@ -724,6 +718,16 @@ def test_InterpolatedStringNode
       RUBY
     end
 
+    def test_concatenated_StringNode
+      pend("PRISM doesn't currently allow to compile this correctly")
```

Comment:
cc @kddnewton, as mentioned on Slack, I don't think we can currently make this test pass with the current parser. It only passed by accident previously.

Dunno if `pend` is the best way, perhaps you'd like an issue open somewhere?